runtime/lua/vim/lsp/handlers.lua

```diff
@@ -333,25 +335,41 @@ M['textDocument/implementation'] = location_handler
 ---     - border:     (default=nil)
 ---         - Add borders to the floating window
 ---         - See |vim.api.nvim_open_win()|
+---     - parametersOnly: (default=false)
+---         - Display only a list of parameters
+---     - suppressPrint: (default=false)
```

Comment:
This option seems out of scope of this PR

Reply:
It seemed like a low hanging fruit, and I saw that the lsp_signature plugin is redefining the same handler just to avoid the prints. Should I revert that particular change?